<launch>
	<!-- Do mapping argument -->
    <arg name="mapping" default="false"/> 

    <!-- Do not use machine files argument -->
    <arg name="sim" default="false"/>

    <!-- Hardware version -->
    <arg name="hardware" default="simulation"/>

 	<node machine="pc1" name="rose_datamanager" pkg="rose_datamanager" type="rose_datamanager" respawn="false"></node>
 	
  	<node machine="pc1" name="rose_operation_manager" pkg="rose_operation_manager" type="rose_operation_manager" output="screen" respawn="false"></node>
 	<node machine="pc1" name="rose_parameter_manager" pkg="rose_parameter_manager" type="rose_parameter_manager" respawn="false"></node>
 	<node machine="pc1" name="rose_action_planner" pkg="rose_action_planner" type="rose_action_planner" output="screen"></node>
 	<node machine="pc1" name="pose_explorer" pkg="rose_pose_explorer" type="rose_pose_explorer" output="screen"></node>
 	<!-- <node pkg="tf" type="static_transform_publisher" name="base_link_lift_transform" args="0.02 0 0.55 0 0 0 /base_link /lift_link 100"/> -->

 	<node machine="pc1" name="soundplay_node" pkg="sound_play" type="soundplay_node.py"></node> 
 	<!-- <node machine="pc1" name="wireless_watchdog_node" pkg="wireless_watchdog" type="watchdog.py"></node>  -->
</launch>
